packages/drivers/odsp-driver/src/fetchSnapshot.ts

```diff
@@ -556,7 +565,8 @@ export async function downloadSnapshot(
             headers.accept = `application/ms-fluid; v=${currentReadVersion}`;
             break;
         default:
-            headers.accept = "application/json";
```

Comment:
how about this (two cases together):

```ts
switch (snapshotFormatFetchType) {
        default:
        case SnapshotFormatSupportType.JsonAndBinary:
            headers.accept = `application/json, application/ms-fluid; v=${currentReadVersion}`;
            break;
        case SnapshotFormatSupportType.Binary:
            headers.accept = `application/ms-fluid; v=${currentReadVersion}`;
            break;
}
```

You can remove SnapshotFormatSupportType.JsonAndBinary case at all - it does not add any value.